internal/lib/dynamic_client.go

```diff
@@ -109,6 +120,34 @@ func GetDynamicInformers() *DynamicInformers {
 	return dynamicInformerInstance
 }
 
+var bootstrapCRFound bool
+
+func BootstrapCRPresent() bool {
+	if bootstrapCRFound {
```

Comment:
In what condition does this `if` block return `true`? It seems as though the `BootstrapCRPresent` method is only getting called once during bootup?